conda/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
     - PYTHON_BUILD_VERSION
     - PYTHON_BUILD_LOCATION
     - TS_CONFIG_MTTCS_DIR
-    - LSST_DDS_DOMAIN
+    - LSST_DDS_PARTITION_PREFIX
```

Comment:
you can actually remove this altogether.. we are using `salobj.BaseCscTestCase` for the CSC unit tests, which creates a random `LSST_DDS_PARTITION_PREFIX` each time you call `async with self.make_csc`, like [here](https://github.com/lsst-ts/ts_m2/blob/7c5aaa5d172b00aaa1777ad377c0ff01ebeeaa22/tests/test_csc.py#L49)

Reply:
Done in commit: b085919. Thanks!